apps/data-pipeline/ap-exams-importer/src/names.ts

```diff
@@ -0,0 +1,21 @@
+// https://apstudents.collegeboard.org/courses
+
+export default {
+  "AP CALCULUS AB": "AP Calculus AB",
+  "AP CALCULUS AB SUB": "AP Calculus BC, Calculus AB subscore",
```

Comment:
I think this can and should be stated more concisely as "AP Calculus BC (AB subscore)". However, open to arguments for keeping it this way.

Reply:
The [College Board page](https://apstudents.collegeboard.org/about-ap-scores/special-score-structure-calculus-bc) calls it the
> Calculus AB subscore on the Calculus BC Exam

or the

> Calculus AB subscore

so I want to be as conservative as possible about how we put the AB and BC aspects together.